packages/dashboard/e2e/wdio.conf.js

```diff
@@ -124,7 +123,14 @@ exports.config = {
   // Services take over a specific job you don't want to take care of. They enhance
   // your test setup with almost no effort. Unlike plugins, they don't add new
   // commands. Instead, they hook themselves up into the test process.
-  services: ['chromedriver'],
+  services: [
+    [
+      'chromedriver',
+      {
+        args: ['--whitelisted-ips='],
```

Comment:
I wanted to get rid of this error:

`bind() failed: Cannot assign requested address (99)`

but it seems like the webdriver is just looking out for ipv6 addresses and then defaulting to ip4 when it can't find it. It doesn't affect the tests.